doc/build/kconfig/preprocessor-functions.rst

```diff
@@ -75,11 +74,49 @@ while the ``*_hex`` version returns a hexadecimal value starting with ``0x``.
    $(dt_nodelabel_reg_size_hex,<node label>[,<index>,<unit>])
    $(dt_nodelabel_reg_size_int,<node label>[,<index>,<unit>])
    $(dt_path_enabled,<node path>)
+
+
+Integer functions
+*****************
+
+The functions listed below can be used to do arithmetic operations
+on integer variables, such as addition, subtraction and more.
```

Comment:
it feels like the blurb from kconfigfunctions.py (" if there is only one argument, it will be split with a comma and processed as a sequence of number") would be useful to have here? I was originally confused that there was only 1 required `value` for functions such as add and thought this was a mistake

Reply:
It does state "See the Python docstrings in [scripts/kconfig/kconfigfunctions.py](https://github.com/zephyrproject-rtos/zephyr/blob/main/scripts/kconfig/kconfigfunctions.py) for detailed documentation." 😆